packages/components/src/pages/VsCodeExtension.vue

```diff
@@ -1209,6 +1214,23 @@ export default {
     updateFilters(updatedFilters) {
       this.$store.commit(SET_SAVED_FILTERS, updatedFilters);
     },
+
+    analysisFindingSelection(findingObject) {
+      const finding = findingObject.resolvedFinding && findingObject.resolvedFinding.finding;
+      if (!finding) return;
+
+      if (finding.checkId === 'n-plus-one-query') {
+        this.$store.commit(SET_HIGHLIGHTED_EVENTS, finding.relatedEvents);
+        this.$store.commit(SET_VIEW, VIEW_FLAMEGRAPH);
```

Comment:
If it's a `Performance` finding go to the flame graph, otherwise go to the trace view.

Reply:
Let's try it